src/js/binaryen.js-post.js

```diff
@@ -2177,6 +2177,23 @@ function wrapModule(module, self) {
       );
     });
   };
+  self['getFunctionTable'] = function() {
+    return {
+      'imported': Boolean(Module['_BinaryenIsFunctionTableImported'](module)),
+      'segments': (function() {
+        var arr = [];
+        for (var i = 0, numSegments = Module['_BinaryenGetNumFunctionTableSegments'](module) ; i !== numSegments ; ++i) {
```

Comment:
Can you remove the spaces before the semicolons in this loop and the loops in the C and JS tests as well? Otherwise LGTM! Nice job :+1:

Reply:
Done. Thanks a lot 👍